src/driver/test/RunCommand.yml

```diff
@@ -34,3 +34,15 @@ Actors:
       OperationName: RunCommand
       OperationCommand:
         currentOp: 1
+- Name: Stepdown
```

Comment:
Just as a heads up, I'm not positive what the intention is for these example workloads, but the stepdown command will probably prevent this one from being run on a standalone with success.